src/EnergyPlus/DualDuct.cc

```diff
@@ -316,7 +316,7 @@ namespace DualDuct {
                                                                                                   cAlphaFields(5));
 
                 state.dataDualDuct->dd_airterminal(DDNum).MaxAirVolFlowRate = NumArray(1);
-                state.dataDualDuct->dd_airterminal(DDNum).ZoneMinAirFracDes = 0.0;
+                state.dataDualDuct->dd_airterminal(DDNum).ZoneMinAirFracDes = 1.0;
```

Comment:
Ran a local regression suite with annual simulations and monthly output. Somehow this change is impacting dual duct even through there is no reheat coil. DDAutoSize shows *no* sizing diffs but the annual energy use is way up, district heating went from 76 to 306 GJ.

Reply:
That's not a good sign. I did not focus on DD when making this change, just thought if the SD TUs needed this change then so goes the dual duct TUs. I will have to review how the ZoneMinAirFracDes is used in DD sizing. This is why unexplained changes in sizing are NEVER merged with develop until there is full consensus from the team.